ttnn/cpp/ttnn/operations/loss/loss_pybind.hpp

```diff
@@ -0,0 +1,131 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include <pybind11/operators.h>
```

Comment:
@KalaivaniMCW please split into .hpp and .cpp and follow ccl/all_gather as an example. There .hpp has a single lightweight include

Reply:
thanks for the reference! I've split pybind file into cpp and hpp